website/static/js/osfLanguage.js

```diff
@@ -80,5 +80,10 @@ module.exports = {
         dataSendError: 'Error sending data to the server: check that all fields are valid, or contact ' + SUPPORT_LINK + ' if the problem persists.',
         creationSuccess: 'Successfully registered new application',
         dataUpdated: 'Application data updated'
+    },
+    projectSettings: {
+        updateSuccessMessage: 'Category, title, and description updated successfully',
+        updateErrorMessage: 'Error updating project settings',
```

Comment:
At a minimum, `osfLanguage.js` defines a variable called `REFRESH_OR_SUPPORT` which can and should be added to the above message to give users more guidance (see example below).

https://github.com/abought/osf.io/blob/develop/website/static/js/osfLanguage.js#L78

If the request is generating a 500 error, that's a whole different ticket to file; most of the time I'd expect 400 error instead (validation / bad request). So long as front end validation warns about most ordinary mistakes, any string that slips past the 400 error would be pretty weird... in which case I wouldn't feel a strong obligation to make the user-facing error message too helpful. (if they want to submit HTML-only project titles, that person is probably Not Our Friend)